lms/djangoapps/teams/static/teams/js/collections/topic.js

```diff
@@ -0,0 +1,24 @@
+;(function (define) {
+    'use strict';
+    define(['common/js/components/collections/paging_collection', 'teams/js/models/topic', 'gettext'],
+        function(PagingCollection, TopicModel, gettext) {
+            var TopicCollection = PagingCollection.extend({
+                initialize: function(topics, options) {
+                    PagingCollection.prototype.initialize.call(this);
+
+                    this.course_id = options.course_id;
+                    this.perPage = topics.results.length;
+                    this.server_api['course_id'] = function () { return encodeURIComponent(this.course_id); };
+                    this.server_api['order_by'] = function () { return this.sortField; };
+                    delete this.server_api['sort_order']; // Sort order is not specified for the Team API
+
+                    this.registerSortableField('name', gettext('name'));
+                    this.registerSortableField('team_count', gettext('team count'));
+                    this.sortField = 'name';
```

Comment:
Note to reviewers: we're not using the `setSortField` setter method because it makes a request to the server, and we bootstrap the `topic` collection with JSON passed in via script tags.